security/dice/src/Microsoft.Azure.Devices.Provisioning.Security.Dice.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
```

Comment:
Is Dice the right name here, x509 is what needs to be supported first and foremost, Dice is an extension to that

Reply:
Naming is still TBD. This library will support the DICE protocol only.
To support X509, an application must inherit from `SecurityClientX509Certificate` and implement the `GetAuthenticationCertificate()` method. This API is currently placed in the Shared assembly.